torch/csrc/jit/mobile/parse_bytecode.cpp

```diff
@@ -24,7 +24,7 @@ IValue expect_field(
       expected_name,
       " found ",
       row->elements().at(0).toStringRef());
-  return std::move(row->elements().at(1));
+  return std::move(row)->elements().at(1);
```

Comment:
I want to remove other std::move also, but due to incomplete support of C++17 on gcc-7, they may fail

Reply:
We have a PR to drop GCC-7 support for this reason.

Reply:
I see, may be we can wait until it is merged.

Reply:
See commit d02959e

Reply:
We can't just return the variable because its type is not the return type and clang says 'local variable 'var' will be copied despite being returned by name'